src/index.js

```diff
@@ -53,6 +54,10 @@ const Builder = ({
           pagination={isPagination ? pages[currentPage].fields : []}
           logic={isLogicOn ? logic : []}
         />
+        {isSingleField && <div className='boomForm-paginationCount'>
+          {currentPage + 1}/{pages.length || 1}
+        </div>
+        }
```

Comment:
This should be shown when the pagination is turned on too

Reply:
Let's change isSingleField to smth like pageCounter. 'Cause builder don't have a clue what is single Field